packages/local_auth/CHANGELOG.md

```diff
@@ -28,8 +37,10 @@
 
 ## 0.6.3+1
 
+* Add `authenticate` method that uses biometric authentication, but allows users to also use device authentication - pin, pattern, passcode.
```

Comment:
We are not going to publish this as all changes are now done in the null-safety version. Please move to the CHANGELOG description above.

Reply:
Moved.